### ISSUE TITLE
🐞 Bug report: Alt image -> testimonial-carousel

### ISSUE THREAD

**Issue by ciltocruz** [elementor/elementor#17680]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

This thread is a bug + new feature request.

I have detected an error regarding SEO with the "testimonial-carousel" widget.

The displayed ALT attribute matches the NAME field.![image](https://user-images.githubusercontent.com/19903919/152153457-0b1ab185-0e7f-4880-a809-136e6db26a4a.png)

In my opinion, for those of us who are interested in SEO, it should show the alt tag of the image if it exists.
If it doesn't exist, I think it's a great idea to show the NAME field.

But the alt tag that we display for that image in wordpress should take precedence.

Thanks!

### Steps to reproduce

1. Upload image to wordpress.
2. Add alt to that image.
3. Go to the Elementor Builder
4. Add carousel testimonials module.
5. Upload an image
6. See source code of the final result.
The ALT tag will not be the one written in wordpress.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

Irrelevant

**Comment by Bovelett** [comment]
This is also crucial to accessibility.

**Comment by rami-elementor** [comment]
You are correct, this is an inconsistent behavior. The "Testimonial" widget uses the image alt text while the "Testimonial Carousel" widget uses the name field as an image ALT.

We plan to fix this inconsistent behavior in v3.16.

**Comment by ciltocruz** [comment]
Great!

**Comment by nicholaszein** [comment]
Hello there!

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
